src/content/roadmap/scaling/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Scaling Ethereum
+description: Ethereum is scaled using rollups. Rollups batch transactions together offchain and send the result to Ethereum.
+lang: en
+image: ../../../assets/eth.png
+alt: "Ethereum roadmap"
+template: roadmap
+---
+
+Ethereum is scaling through [rollups](/layer-2/#rollups), which batch transactions together and send the output to Ethereum. Even though rollups are up to eight times less expensive than Ethereum Mainnet, it's possible to optimize rollups further to reduce costs for end users. Rollups also rely on some centralized components that developers can remove as the rollups mature.
+
+<InfoBanner mb={8}>
+  <h4 style="margin-top: 0">Transaction costs</h4>
```

Comment:
Not blocking, but this creates incorrect heading order by using an H4 here and is an accessibility issue.

cc: @corwintines